### DISCUSSION TITLE
Board Arrived Today!

### DISCUSSION THREAD

**Discussion by TwoPigsToTheWind** [slu4coder/Minimal-64x4-Home-Computer#16] [General]
Super excited to work with a real, physical Minimal. I ended up ordering the standard size board and not the microATX variant I was considering. Looking forward to getting it all soldered up and then working on a sound card. 

Anyone have any interesting case ideas? I think a retro looking terminal style case would be pretty sweet and fit the black and white text based vibe of the Minimal pretty well. You can still get ps/2 control chips in DIP format, so building a keyboard into the case should be doable.  Having a built in monitor might be more trouble than it's worth though. Moving just a little bit forward in time and simplifying things by ditching the monitor, an Apple II or C64 style case would similarly have a nice retro vibe to it. One is more expansion friendly than the other of course.

 Ultimately I'm going to hold off on any reformatting of the main board or building/ordering any type of case till  I have a better idea of how useful potential expansion cards will be in general to help figure the number of slots that might be needed, and what types of size, power, and I/O requirements cards I or someone else develops will have.

**Comment by nateo87** [comment]
Curious what ICs you found that are through hole PS/2 encoders! I've always been enamored with the design of the Ohio Scientific Challenger series, and would love to design something akin to that with an integrated keyboard. I know whatever type of case I design, I'd want the main board of the Minimal to be presentable, whether that's with a clear top, or a lid that can be lifted to display the star of the show.

**Reply by TwoPigsToTheWind** [reply]
Chips like the HT82K28A will pop up on ebay from time to time. An Arduino Nano or other MC could also be used to generate the ps/2 signal, I don't necessarily think that would take away from the Minimal's 'no MCs'  guideline since the Minimal itself handles ps/2 just fine and this would be more of a case of using a MC to build a keyboard controller and not using a MC to read keyboard inputs. 

I like the look of Ohio Scientific's Challenger series, Reminds me of the Sol-20. Looking in a more terminal like direction I'm a fan of the VT05, VT52 and ADM-3A

**Reply by nateo87** [reply]
Ooo, the VT05 would be pretty.
The HT82K28A seems like a great choice, and plentiful enough despite being out of production. 

Looking forward to that sound card!

**Reply by nateo87** [reply]
Hm, this seems to be an OK option to be able to use smaller, open source USB keyboards that translate to a PS/2 interface. But the thought of using a Pi Pico sure seems like overkill...
https://github.com/No0ne/ps2x2pico?tab=readme-ov-file

**Reply by TwoPigsToTheWind** [reply]
A Pico or other micro controller would certainly do the trick, could use the Nano off of the Flash Programmer. As long as the micro controller is fast enough to scan thru the keyboard matrix at a good clip and generate the appropriate ps/2 serial output.   

Sound card hopefully isn't too far off. I need to inventory my bins of parts and see if there are any odds and ends I need to order for the main Minimal board, as an example I know I don't have the right VGA header. After there's a firm list of parts to order for that I'll tack on the chips for the sound card for the sake of saving on shipping.
